openui5/tsconfig.json

```diff
@@ -2,16 +2,10 @@
     "compilerOptions": {
         "module": "commonjs",
         "target": "es6",
-        "noImplicitAny": false,
+        "noImplicitAny": true,
         "strictNullChecks": false,
-        "types": [],
+        "types": ["jquery"],
```

Comment:
Use this:

```json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es6",
        "noImplicitAny": true,
        "strictNullChecks": false,
        "baseUrl": "../",
        "typeRoots": [
            "../"
        ],
        "types": [],
        "noEmit": true,
        "forceConsistentCasingInFileNames": true
    },
    "files": [
        "jquery.sap.d.ts",
        "sap.m.d.ts",
        "sap.ui.d.ts",
        "ui5-tests.ts"
    ]
}
```

And at the top of `jquery.sap.d.ts` (after the header), write `/// <reference types="jquery" />`.

On another note, this package has no `index.d.ts`. So it must be used like `/// <reference types="openui5/sap.ui" />`.
If there were an `index.d.ts`, it could be automatically referenced.
If people are likely to need both `sap.m` and `sap.ui`, there should be an index referencing both of those (with `/// <reference path="./sap.m.d.ts" />`).
Also, only the index will need a header.